starter-code/pizza.js

```diff
@@ -1 +1,96 @@
 // Write your Pizza Builder JavaScript in this file.
+$( document ).ready(function(){
+
+  var total = 13;//default
+  $('strong').text(total);
+  var priceMushroom=1;
+  var pricePepperoni=1;
+  var priceGreenPeppers=1;
+  var priceSauce=3;
+  var priceCrust=5;
```

Comment:
I like that you defined variables for that; it's very good style. You can even put that in an object. Example:

```
var prices = {
  mushroom: 1,
  pepperoni: 1,
  greenPeppers: 1,
  sauce: 3,
  crust: 5
}
```